server/migrations/20230429013050-modify_organization_add_mfa_field.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    await queryInterface.addColumn('organization', 'ismfaenabled', {
+      type: Sequelize.DataTypes.BOOLEAN,
+      defaultValue: false,
+      allowNull: false,
+    });
+  },
+
+  async down(queryInterface, Sequelize) {
+    /**
+     * Add reverting commands here.
+     *
+     * Example:
+     * await queryInterface.dropTable('users');
+     */
+    await queryInterface.removeColumn('ismfaenabled');
```

Comment:
I believe this is incorrect- you're missing the tablename `'organizations'` as the first paramater to removeColumn.

You can test the "down" migration by logging into the container, changing into the server directory, and running: `sequelize db:migrate:undo`. It should cleanly undo the changes in the `up` (not always possible, but generally speaking), such that you could run `sequelize db:migrate` again to run it again without problem.

Reply:
makes sense. Thanks. 
I was getting this error initially so I guess undefined here would be referring to the lack of 'organization' table. 
![image](https://github.com/sfbrigade/bats-server/assets/43530539/56e6cab5-9fc8-48bc-8393-ad1b23a40372)